supabase/migrations/20260505000002_fix_082_topic_sort_order.sql

```
@@ -0,0 +1,22 @@
+
+UPDATE easa_topics
+SET sort_order = v.sort_order
+FROM (VALUES
+  ('082-01', 11),
+  ('082-02', 12),
+  ('082-03', 13),
+  ('082-04', 14),
+  ('082-06', 16)
+) AS v(code, sort_order)
+WHERE easa_topics.code = v.code
+  AND easa_topics.subject_id = (SELECT id FROM easa_subjects WHERE code = '080');
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)